packages/server-web/src/client/component/Home.vue

```diff
@@ -19,6 +19,11 @@ export default {
     "todo-list": TodoList,
     "todo-text": TodoText
   },
+  data() {
+    return {
+      commands: this.$store.getters
```

Comment:
does this return all getters or is there any JS magic involved? If it's all the getters, a different variable name might be better suited?

Reply:
Seems to be some bug here. I saw that I should get it in the script editor itself anyways, so not getting it through `$props` to script-editor anymore, making this here obsolete.